test/acceptance/kafo_configure_test.rb

```diff
@@ -173,5 +173,41 @@ module Kafo
         _(File.exist?("#{INSTALLER_HOME}/testing")).must_equal true
       end
     end
+
+    describe 'the configuration file' do
+      before do
+        @saved_values = Kafo::Configuration::DEFAULT.reject do |key, value|
+          value.nil?
+        end
+      end
+
+      it 'must only save defined values' do
+        code, _, err = run_command '../bin/kafo-configure'
+
+        _(code).must_equal 0, err
+        _(YAML.load_file(KAFO_CONFIG).keys.sort).must_equal @saved_values.keys.sort
+      end
+
+      it 'must not save non persisted command line options' do
+        code, _, err = run_command '../bin/kafo-configure --noop'
```

Comment:
Could you also add colors here, similar to the persisted command line options?

Reply:
Check now.. is that what you meant?